src/Stepper.tsx

```diff
@@ -0,0 +1,54 @@
+import React, { memo, forwardRef } from "react";
+import { symToStr } from "tsafe/symToStr";
+import { assert } from "tsafe/assert";
+import type { Equals } from "tsafe";
+import { fr } from "./lib";
+import { cx } from "./lib/tools/cx";
+
+// We make users import dsfr.css, so we don't need to import the scoped CSS
+// but in the future if we have a complete component coverage it
+// we could stop requiring users to import the hole CSS and only import on a
+// per component basis.
+import "./dsfr/component/stepper/stepper.css";
+
+export type StepperProps = {
+    className?: string;
+    currentStep: number;
+    steps: number;
+    title: string;
+    nextTitle?: string;
+};
+
+/** @see <https://react-dsfr-components.etalab.studio/?path=/docs/components-stepper> */
+export const Stepper = memo(
+    forwardRef<HTMLDivElement, StepperProps>((props, ref) => {
+        const { className, currentStep, steps, title, nextTitle, ...rest } = props;
+
+        assert<Equals<keyof typeof rest, never>>();
+
+        return (
+            <div className={cx(fr.cx("fr-stepper"), className)} ref={ref}>
+                <h2 className="fr-stepper__title">
+                    <span className="fr-stepper__state">
+                        Étape {currentStep} sur {steps}
```

Comment:
@revolunet would you internationalize this?

Reply:
yep

Reply:
Pour "Etape 1 sur 3" on fait comment ? je n'ai pas l'impression qu'on gère l'interpolation de variables ?

Reply:
Tu as raison je vais étendre le modèle

Reply:
@revolunet Maintenant c'est possible: https://github.com/codegouvfr/react-dsfr/blob/2fdff992c711edeeec8ce14087e26f3a17adb8bb/src/Stepper.tsx#L66-L67